sasstests.py

```diff
@@ -846,6 +846,10 @@ def write_file(filename, contents):
 
 class CompileDirectoriesTest(unittest.TestCase):
 
+    def test_directory_does_not_exist(self):
+        with self.assertRaises(OSError):
+            sass.compile(dirname=('i_dont_exist_lol', 'out'))
```

Comment:
According to the docs, use as a context manager of [`assertRaises()`][1] is introduced in Python 2.7.  See also [py26 build][2].

[1]: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises
[2]: https://travis-ci.org/dahlia/libsass-python/jobs/106892363